hack/dev-setup-local

```diff
@@ -98,9 +98,6 @@ kubectl config use-context gardener-dev \
 # Different base64 implementations have different flags
 LOCAL_SEED_KUBECONFIG_B64=$(base64 ${LOCAL_SEED_KUBECONFIG} | tr -d '\r\n')
 
-sed -e "s/name: my-seed/name: local/g" example/25-controllerinstallation.yaml | \
```

Comment:
Why is that removed? If we need to remove it, can you please adapt the local dev. setup documentation accordingly?

Reply:
It is removed because now there's a separated step for extension installation. Therefore, this installation is not needed. No documentation change is required because this setup was executed implicitly (not documented).